sway/desktop/render.c

```diff
@@ -342,15 +342,15 @@ void render_blur(bool optimized, struct sway_output *output,
 // _box.x and .y are expected to be layout-local
 // _box.width and .height are expected to be output-buffer-local
 void render_box_shadow(struct sway_output *output, pixman_region32_t *output_damage,
-		const struct wlr_box *_box, const float color[static 4],
-		float blur_sigma, float corner_radius) {
+		const struct wlr_box *_box, const float color[static 4], float blur_sigma, 
+		float corner_radius, float offset_x, float offset_y) {
 	struct wlr_output *wlr_output = output->wlr_output;
 	struct fx_renderer *renderer = output->renderer;
 
 	struct wlr_box box;
 	memcpy(&box, _box, sizeof(struct wlr_box));
-	box.x -= blur_sigma;
-	box.y -= blur_sigma;
+	box.x -= blur_sigma - offset_x;
+	box.y -= blur_sigma - offset_y;
```

Comment:
Q: wouldn't we want to add the offsets?

Reply:
When adding the offsets, the box gets moved into the opposite direction.

Reply:
Ah yes, missed the double negative of subtracting from a subtracted value